torch/distributed/rpc/__init__.py

```diff
@@ -165,6 +166,10 @@ def init_rpc(
             # If world_size is not set in construction and also not set in environment variables
```

Comment:
nit: this is prior to this PR. In the comment above:

"Create store, performs rendezvous" -> "Create store, perform rendezvous"?